src/components/ec-radio-btn/ec-radio-btn.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+  <div
+    class="ec-radio-btn"
+    :class="[$attrs.class]"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-radio-btn ec-radio-btn` : `ec-radio-btn ec-radio-btn`"
+    @click="onRadioBtnClick();"
+  >
+    <input
+      v-bind="{
+        'aria-describedby': errorId,
```

Comment:
We should add here the label and the description too.

Reply:
i cannot see these attributes in the native radio

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby#:~:text=The%20global%20aria%2Ddescribedby%20attribute%20identifies%20the%20element%20(or%20elements)%20that%20describes%20the%20element%20on%20which%20the%20attribute%20is%20set. `label` and `description` are elements in our component, we can use list of elements with `aria-describedby`